pkgs/development/libraries/libhttpseverywhere/default.nix

```diff
@@ -19,7 +19,7 @@ in stdenv.mkDerivation rec {
 
   doCheck = true;
 
-  checkPhase = "./httpseverywhere_test";
+  checkPhase = "cd test && ./httpseverywhere_test && cd ..";
```

Comment:
I think more typical is `(cd test && ./httpseverywhere_test)` so you don't need to `cd` back.

Reply:
thanks, that look better.